### DISCUSSION TITLE
New question types

### DISCUSSION THREAD

**Discussion by justynpride** [HiEventsDev/Hi.Events#107] [Ideas]
amazing application. a couple of thoughts on additional question types.
**Date** it would be great to have a date field question type. I'd like to be able to ask for a date of birth for instance and collect it in a standard format, and not single text format. I’ve had need to restrict tickets to those within a certain age bracket - think kids groups. To be able to also specify and upper and lower age restriction that therefore only allows the correct age range to be selected to displayed would be amazing.
**Fileupload** some events may require some documentation on a ticket - say proof of safeguarding checks for working im a kids group, or submission of a photo for a volunteer.

**Comment by daveearley** [comment]
Nice suggestions! I'll add Date to the list soon. I'll have to think about file uploads. While I think it's a good idea, there are a few logistical and privacy issues related to storing attendees documentation that would need to be ironed out.

**Reply by justynpride** [reply]
@daveearley I concor on the upload side of things. Alongside the clarity of GDPR, it almost needs an individual check box on the upload to confirm that you have permission to upload and agree with the organisations gdpr policy.
